webpack/scenes/Settings/Tables/TableActions.js

```diff
@@ -0,0 +1,73 @@
+import { foremanApi, userId } from '../../../services/api';
+
+import {
+  TABLES_REQUEST,
+  TABLES_SUCCESS,
+  TABLES_FAILURE,
+  CREATE_TABLE,
+  CREATE_TABLE_SUCCESS,
+  CREATE_TABLE_FAILURE,
+  UPDATE_TABLE,
+  UPDATE_TABLE_SUCCESS,
+  UPDATE_TABLE_FAILURE,
+} from './TableConstants';
+
+const getResponseError = ({ data }) => data && (data.displayMessage || data.error);
+
+export const loadTables = () => (dispatch) => {
+  dispatch({ type: TABLES_REQUEST, params: {} });
+  // foremanApi.delete(`/users/${userId}/table_preferences/Katello::Subscriptions`, {});
```

Comment:
Some leftover?

Reply:
I updated the branch and removed extra comment